File: Book.java
import java.util.Random;
public class Book {
    private String title;
    private boolean isRented = false;
    private int id=0;
    Random random = new Random();

    public Book(String title) {
        this.id = random.nextInt(1,1000);
        this.title = title;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }


    public boolean isRented() {
        return isRented;
    }

    public void setRented(boolean isRented) {
        this.isRented = isRented;
    }

    


    
}

File: Book.py
import random
class Book:
    __title=""
    __isRented = False
    __id=0
    
    def __init__(self,title):
        self.__title = title
        
    def get_title(self):
        return self.__title
    
    def set_title(self, title):
        self.__title = title
        
    def is_rented(self):
        return self.__isRented
    
    def set_rented(self, isRented):
        self.__isRented = isRented
        